src/java/htsjdk/variant/variantcontext/GenotypesContext.java

```diff
@@ -660,6 +660,7 @@ public boolean containsSample(final String sample) {
         return sampleNameToOffset.containsKey(sample);
     }
 
+    /** return true if all {@link GenotypesContext#getSampleNames() contains all sample */
```

Comment:
Wording is awkward here -- "return true if the list of samples in this context as returned by `{@link #getSampleNames()}` contains all samples in the Collection passed in"